paper/paper.md

```diff
@@ -6,102 +6,102 @@ tags:
   - neuroimaging
 authors:
   - affiliation: 1
-    name: Yarkoni, Tal
+    name: Tal Yarkoni
     orcid: 0000-0002-6558-5113
   - affiliation: 2
-    name: Markiewicz, Christopher J.
+    name: Christopher J Markiewicz
     orcid: 0000-0002-6533-164X
   - affiliation: 1
-    name: de la Vega, Alejandro
+    name: Alejandro I de la Vega
```

Comment:
vim?

```suggestion
    name: Alejandro de la Vega
```

Reply:
Middle initial, but it looks like @adelavega doesn't actually publish with it, so I accept your suggestion. :)

Reply:
heh yeah. probably should have but now its a little too late

Reply:
I didn't even read it as an initial I, but thought "the first". Vim was only my second attempt at a likely explanation.